homeassistant/components/wink.py

```diff
@@ -56,9 +58,32 @@
     'client_secret': 'CLIENT_SECRET_HERE'
 }
 
-SERVICE_ADD_NEW_DEVICES = 'add_new_devices'
+SERVICE_ADD_NEW_DEVICES = 'pull_newly_added_devices_from_wink'
```

Comment:
Pull seems an odd pick. What about `refresh_devices`? Nah, that's ambiguous with updating states. Let's leave it as this.